.github/lib/discussions.rb

```diff
@@ -8,7 +8,10 @@
   :id,
   :url,
   :title,
-  :labelled
+  :labelled,
+  :body,
+  :created_at,
+  :is_answered
 ) do
```

Comment:
by adding `:is_answered` to the struct, I eliminate the need for an `is_answered?` lookup, reducing the overall calls needed for the `close_incident_discussions` action